app/application.py

```diff
@@ -67,19 +67,33 @@ def convert(link, model):
         return
 
 
-def write_to_file(result, url, title, date):
+def write_to_file(result, url, title, date, tags, category, speakers):
     transcribed_text = result[0]
     video_title = result[1][:-4]
     if title:
         file_title = title
     else:
         file_title = video_title
-
+    tags = tags.strip()
+    tags = tags.split(",")
+    for i in range(len(tags)):
+        tags[i] = tags[i].strip()
+    speakers = speakers.strip()
+    speakers = speakers.split(",")
+    for i in range(len(speakers)):
+        speakers[i] = speakers[i].strip()
+    category = category.strip()
```

Comment:
this line crashed on me when I didn't pass in a category

<img width="637" alt="Screen Shot 2023-01-18 at 8 15 01 PM" src="https://user-images.githubusercontent.com/755825/213333230-8bd7ed2d-11fe-463a-a7bc-aa3b7296a1b0.png">